components/eamxx/src/share/io/scream_scorpio_interface.F90

```diff
@@ -575,7 +575,7 @@ function get_variable_metadata_float(filename, varname, metaname) result(metaval
     ! Find the pointer for this file
     call lookup_pio_atm_file(trim(filename),pio_file,found)
     if (.not.found ) then
-      call errorHandle("PIO ERROR: error getting metadata for variable "//trim(varname)//" in file "//trim(filename)//".\n PIO file not found or not open.",-999)
+      call errorHandle("PIO ERROR: error getting metadata for file "//trim(filename)//".\n PIO file not found or not open.",-999)
```

Comment:
Why removing the var name in the message? It seemed useful.

Reply:
It's true that the issue is that a file is not found, but knowing what var was pinged imho can help us to track where in the code this error is originating. For instance, this can be helpful if there are two streams for the same file (like pg2 and dyn streams).

Reply:
sounds good, I'm happy to revert this change.